include/picongpu/CMakeLists.txt

```diff
@@ -254,6 +254,9 @@ set(PIC_VERBOSE "1" CACHE STRING
     "Set verbosity level for PIConGPU (default is only physics output)")
 add_definitions(-DPIC_VERBOSE_LVL=${PIC_VERBOSE})
 
+set(PIC_TRACE_OPENPMD_TIMES "0" CACHE STRING "Set the maximum number of MPI ranks that create a file with trace information.")
```

Comment:
Maybe add something like "NUMBER_OF_FILES" to this option name so people won't think this is a simple boolean